data/actions/scripts/other/bed_modification_kits.lua

```diff
@@ -1,23 +1,23 @@
-local BEDS = {
+local beds = {
 	[7904] = {{7811, 7812}, {7813, 7814}}, -- green kit
 	[7905] = {{7819, 7820}, {7821, 7822}}, -- yellow kit
 	[7906] = {{7815, 7816}, {7817, 7818}}, -- red kit
-	[7907] = {{1754, 1755}, {1760, 1761}},  -- removal kit
+	[7907] = {{1754, 1755}, {1760, 1761}}, -- removal kit
 	[20252] = {{20197, 20198}, {20199, 20200}} -- canopy kit
 }
 
-local function internalBedTransform(item, targetItem, toPosition, ids)
-	targetItem:transform(ids[1])
+local function internalBedTransform(item, targetItem, tile, toPosition, itemArray)
```

Comment:
That was because I used the old tile, but Ninja pointed out that the position is changed. Forgot to change that when I reverted the tile userdata itself.